src/cadence/mainnet/getNFTs.cdc

```diff
@@ -520,7 +520,7 @@ pub fun getEverbloom(owner: PublicAccount, id: UInt64): NFTData? {
             storage_path: "Everbloom.CollectionStoragePath",
             public_path: "Everbloom.CollectionPublicPath",
             public_collection_name: "Everbloom.PrintCollectionPublic",
-            external_domain: ""
+            external_domain: "https://web.everbloom.app/"
```

Comment:
Thanks for submitting this change! Can you update the testnet folder as well?

Reply:
Is it required? because we still have not set up our testnet token URI. This is a temporary fix. We will update our smart contracts according to the new Flow NFTs standard soon.